bigtable/tests/system.py

```diff
@@ -154,6 +162,35 @@ def test_create_instance(self):
 
         self.assertEqual(instance, instance_alt)
         self.assertEqual(instance.display_name, instance_alt.display_name)
+        # Make sure that by default a PRODUCTION type instance is created
+        self.assertIsNone(instance.instance_type)
+        self.assertEqual(instance_alt.instance_type, _PRODUCTION)
+        self.assertIsNone(instance.labels)
+        self.assertFalse(instance_alt.labels)
+
+    def test_create_instance(self):
+        from google.cloud.bigtable_admin_v2 import enums
+        _PRODUCTION = enums.Instance.Type.PRODUCTION
```

Comment:
Since `PRODUCTION` is the default if `instance_type` is not specified, it would be a better test to use a different type here.

Reply:
Changed to `DEVELOPMENT`. Had to pass serve_nodes. Should be fixed in Cluster.py, serve_nodes should default to None.